.bumpversion.cfg

```diff
@@ -1,10 +1,10 @@
 [bumpversion]
-current_version = 0.1.0
```

Comment:
Why go from 1.0 to 0.1? If the template starts from 0.1.0, I think that makes sense. This is not a bugfix, we started adding features right away.

Reply:
Ok! I'll keep it at 0.1.0.